app/templates/_README.md

```diff
@@ -7,55 +7,45 @@
 _(Requires [wp-cli](http://wp-cli.org/) to download WordPress)_
 ```
 $ git clone
-$ cd <%= appSlug %> && wp core download
-$ npm install && bower install
-$ sh scripts/download-plugins.sh
+$ cd <%= appSlug %> && npm run setup
 ```
 
-### Sync database
-
-**1. Create database in MySQL/phpMyAdmin**
-
-**2. Install [wordpress-db-sync](https://github.com/micjamking/wordpress-db-sync)**
-```
-$ cd db/ && git clone git@github.com:micjamking/wordpress-db-sync.git
-$ cp -R wordpress-db-sync/* ./
-$ rm -rf wordpress-db-sync/
-```
-**3. Configure pull.sh, push.sh, & update.sql with your database name, user, and password**
```

Comment:
@nainoa This part is still required; the `npm run setup` installs wordpress-db-sync, but it doesn't know what your db & mysql credentials are to complete this step.